templates/schedule.html

```diff
@@ -6,21 +6,16 @@
   <style>{% inline_file "/static/css/schedule.css" %}</style>
 {% endblock %}
 
-{% block preload %}
-  <script type="module">
-    import 'https://unpkg.com/@polymer/paper-toggle-button/paper-toggle-button.js?module';
-    import '/static/elements/chromedash-schedule.js';
-  </script>
-{% endblock %}
-
 {% block subheader %}
   <div id="subheader">
     <!--<h2>Chrome release timeline</h2>-->
     <div style="flex:1">
       <h3>Release timeline</h3>
       <p class="description">Please note, all dates are estimates and are subject to change.</p>
     </div>
-    <paper-toggle-button noink>Hide Blink components</paper-toggle-button>
+    <!-- <paper-toggle-button> doesn't working here. Related links:
+      https://github.com/PolymerElements/paper-toggle-button/pull/132 -->
+    <label><input type="checkbox" class="paper-toggle-button">Hide Blink components</label>
```

Comment:
Might want to give the `<input>` an extra 4-8px of right margin to separate it from the label a tad

Reply:
Done